setup_gui.py

```diff
@@ -342,102 +389,116 @@ def add_cm_tab(self, cm_name: str):
         hlayout.addWidget(gui_cryomodule.turn_off_button)
         hlayout.addWidget(gui_cryomodule.abort_button)
         hlayout.addStretch()
-        
+
         vlayout.addLayout(hlayout)
-        
+
         groupbox: QGroupBox = QGroupBox()
         all_cav_layout: QGridLayout = QGridLayout()
         groupbox.setLayout(all_cav_layout)
         vlayout.addWidget(groupbox)
         for cav_num in range(1, 9):
             cav_groupbox: QGroupBox = QGroupBox(f"CM{cm_name} Cavity {cav_num}")
             cav_groupbox.setStyleSheet("QGroupBox { font-weight: bold; } ")
-            
+
             cav_vlayout: QVBoxLayout = QVBoxLayout()
             cav_groupbox.setLayout(cav_vlayout)
             cav_widgets = gui_cryomodule.gui_cavities[cav_num]
-            cav_desamp_hlayout: QHBoxLayout = QHBoxLayout()
-            cav_desamp_hlayout.addStretch()
-            cav_desamp_hlayout.addWidget(QLabel("Amplitude: "))
-            cav_desamp_hlayout.addWidget(cav_widgets.ades_spinbox)
-            cav_desamp_hlayout.addWidget(cav_widgets.aact_readback_label)
-            cav_desamp_hlayout.addStretch()
-            
+            cav_amp_hlayout: QHBoxLayout = QHBoxLayout()
+            cav_amp_hlayout.addStretch()
+            cav_amp_hlayout.addWidget(QLabel("ACON: "))
+            cav_amp_hlayout.addWidget(cav_widgets.acon_label)
+            cav_amp_hlayout.addWidget(QLabel("AACT: "))
+            cav_amp_hlayout.addWidget(cav_widgets.aact_readback_label)
+            cav_amp_hlayout.addStretch()
+
             cav_button_hlayout: QHBoxLayout = QHBoxLayout()
             cav_button_hlayout.addStretch()
             cav_button_hlayout.addWidget(cav_widgets.setup_button)
             cav_button_hlayout.addWidget(cav_widgets.turn_off_button)
             cav_button_hlayout.addWidget(cav_widgets.abort_button)
             cav_button_hlayout.addWidget(cav_widgets.expert_screen_button)
             cav_button_hlayout.addStretch()
-            
-            cav_vlayout.addLayout(cav_desamp_hlayout)
+
+            cav_vlayout.addLayout(cav_amp_hlayout)
             cav_vlayout.addLayout(cav_button_hlayout)
             cav_vlayout.addWidget(cav_widgets.status_label)
-            all_cav_layout.addWidget(cav_groupbox,
-                                     0 if cav_num in range(1, 5) else 1,
-                                     (cav_num - 1) % 4)
+            all_cav_layout.addWidget(
+                cav_groupbox, 0 if cav_num in range(1, 5) else 1, (cav_num - 1) % 4
+            )
```

Comment:
What's cav_groupbox? CM# and Cavity#? Why only 0 or 1? Why cav_num -1?

Reply:
can_groupbox is the QGroupBox that holds all of a cavity's widgets (the labels and buttons and things).
The arguments for all_cav_layout are (widget, row number, column number), which are 0 indexed (whereas cavity numbering is 1 indexed, hence the -1). I only have two rows, so row 0 for rack A and row 1 for rack B.